pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -152,9 +189,20 @@ stdenv.mkDerivation rec {
     "-Donnxruntime_USE_FULL_PROTOBUF=OFF"
   ] ++ lib.optionals pythonSupport [
     "-Donnxruntime_ENABLE_PYTHON=ON"
+  ] ++ lib.optionals cudaSupport [
+    "-DFETCHCONTENT_SOURCE_DIR_CUTLASS=${cutlass}"
```

Comment:
Does their `FetchContent_MakeDeclare` include `FIND_PACKAGE_ARGS`? If not, it's probably a good idea to contact upstream and ask them to


Meantime you could also try https://cmake.org/cmake/help/latest/module/FetchContent.html#variable:FETCHCONTENT_TRY_FIND_PACKAGE_MODE

Reply:
Oh I now realize I had commented about this before 🙈

Reply:
I've yet to look into this, this will need a bit more work than the rest.

Reply:
Ok, let's keep this for later PRs